docs/lib/datastore/fragments/js/data-access/delete-snippet.md

```diff
@@ -5,17 +5,17 @@ const todelete = await DataStore.query(Post, "1234567");
 DataStore.delete(todelete);
 ```
 
-You can also pass predicate operators to delete multiple items. For example will delete all inactive posts:
+You can also pass predicate operators to delete multiple items. For example will delete all draft posts:
 
 ```js
-await DataStore.delete(Post, c => c.status("eq", PostStatus.INACTIVE));
+await DataStore.delete(Post, c => c.status("eq", PostStatus.DRAFT));
```

Comment:
What does c stand for in these examples? If it's copy, maybe we should just spell it out or else it's kind of confusing.

If it''s not copy, maybe we should instead use `p` or lowercase `post`